pyproject.toml

```diff
@@ -1,5 +1,10 @@
 [build-system]
-requires = ["setuptools>=61.0", "BAC0==22.9.21"]
+requires = [
+    "setuptools>=61.0",
+    "BAC0==22.9.21",
+    "pytz==2022.7.1",
+    "decorator==5.1.1",
```

Comment:
Those new dependencies don't seem to be used anywhere. Are those added on purpose or by accident?

Reply:
I added them on purpose. When trying to run your code (after doing `python3 -m pip install .`) I get errors saying that there are missing dependencies. But I'm not a python developer, I might have done something wrong.

Furthermore when running your integration in home assistant dev mode I get the same error. Adding the missing requirements requirements.json seems to fix it.